definitelyTypedTests/index.js

```diff
@@ -161,3 +161,19 @@ createDefinitelyTypedMock<typeof pak>();
             });
         });
 }
+
+function removeWarningLogsForFilesInDifferentDirectories(log, typePath) {
+    const logLines = log
+        .split('\n')
+        .filter((line) => !!line);
+
+    for (let i = logLines.length - 1; i >= 0; i--) {
+        if (/WARNING[^\n]+?: Transformer/.test(logLines[i])) {
+            if (!logLines[i+2].includes(typePath)) {
```

Comment:
If I understood the problem correctly we would like to remove definitely typed errors from other definitely type dependencies? I think there is still the possibility that a definitely typed depends on NON definitely typed types. What would happen in that case?

Reply:
Two alternatives:
- if the package is installed in the definitely typed node_modules then it will still work since the file is in the folder `DefinitelyTyped/types/CURRENT_TYPE/node_modules/***`
- if the package is installed in the root folder then it won't work

You're right... you know what, I'll close this PR, I want to do something different with the warnings in the UI.